receiver/rabbitmqreceiver/metadata.yaml

```diff
@@ -1 +1,65 @@
 name: rabbitmqreceiver
+
+attributes:
+  rabbitmq.queue.name:
+    description: The name of the RabbitMQ queue.
+  rabbitmq.node.name:
+    description: The name of the RabbitMQ node.
+  rabbitmq.vhost.name:
+    description: The name of the RabbitMQ vHost.
+  message.state:
+    value: state
+    description: The state of messages in a queue.
+    enum:
+      - ready
+      - unacknowledged
+metrics:
+  rabbitmq.consumer.count:
+    description: The number of consumers currently reading from the queue.
+    unit: "{consumers}"
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  rabbitmq.message.delivered:
+    description: The number of messages delivered to consumers.
+    unit: "{messages}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  rabbitmq.message.published:
+    description: The number of messages published to a queue.
+    unit: "{messages}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  rabbitmq.message.acknowledged:
+    description: The number of messages acknowledged by consumers.
+    unit: "{messages}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
+  rabbitmq.message.dropped:
+    description: The number of messages dropped as unroutable.
+    unit: "{messages}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    enabled: true
```

Comment:
Are we sure that these cumulative metrics cannot be combined with `state` attribute? Following [OTel guidelines](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#general-guidelines) they can be one metrics if aggregation is _meaningful_. Isn't it the case for these metrics?

cc @djaglowski

Reply:
The reason they are not combined is because they are not exclusive. Meaning a message can both be published and dropped or acknowledged and delivered for example. If we combine them we could get double counting. The aggregation is not meaningful because of this.

Reply:
Ok. Sorry I'm not very familiar with these metrics. If they are combined in a metric with two attributes, can it make avg or/and sum aggregations meaningful?

Reply:
Yeah no worries worth asking. What I meant was I don't think you can combine them at all. None of the states are exclusive from the others so it's possible to get multiple counting of the same message if we do any combination. I think because of this multiple counting it would create an incorrect aggregation.

Reply:
Got it. Thanks for explanation.